noggin/templates/group.html

```diff
@@ -10,25 +10,38 @@
   <div class="container section">
     <div class="row py-4" data-section="identity">
       <div class="col">
-          <h3>{{ group.name }}</h3>
+          <h3 class="mb-0">
+            {{ group.name }}
+            {% if edit_group_details is defined %}
+              <span class="pl-1">{{ edit_group_details()}}</span>
+            {% endif %}
+          </h3>
           {% if group.description %}<div>{{ group.description }}</div>{% endif %}
+          {% if group.mailing_list %}
+            <div><i class="fa fa-fw fa-envelope" aria-hidden="true"></i> <a href="{{group.mailing_list}}">{{group.mailing_list}}</a></div>
+          {% endif %}
+          {% if group.irc_channel %}
+            <div><i class="fa fa-fw fa-comments-o" aria-hidden="true"></i> <a href="{{group.irc_channel}}">{{group.irc_channel}}</a></div>
+          {% endif %}
+      </div>
+      <div class="col-auto align-self-center">
+          {% if group.url %}
```

Comment:
You don't actually need the `if` block, if the list it empty the `for` loop will just produce no output. Unless you want to put the `if` block around the `div`.